src/components/Header.vue

```diff
@@ -1,42 +1,28 @@
 <template>
-    <header>
-        <img alt="a postit" src="../assets/postit.png">La Postiterie
-    </header>
+  <el-header>
+    <img
+      alt="La Postiterie" 
+      src="../assets/logoandtitle.png" 
```

Comment:
Isn't there a plugin in Vue to import the png file as a module? It would be more resilient to refactoring and file moves.

Reply:
Seems like you can do it without a plugin, i'm on it